server/routes/analyticsRouter.ts

```diff
@@ -87,6 +88,8 @@ async function transformAnalyticsError<R>(reportPromise: Promise<R>): Promise<R>
   }
 }
 
+export const cache = new StitchedTablesCache()
```

Comment:
That's nicer, there would still be one instance created and shared across all the request handlers per running process AFAICT.

Reply:
yup, that's exactly the plan: a global cache per-process (there's only 1 process per pod)